src/constants/channels.js

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2016 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import keymirror from 'keymirror';
+
+const ChannelTypes = keymirror({
```

Comment:
I'm fine using keymirror here for now, but we may run into problems that'll require us to namespace these in the future